packages/apputils/src/sanitizer.ts

```diff
@@ -952,11 +958,7 @@ export class Sanitizer implements ISanitizer {
     // 'src' Attributes are validated to be URIs, which does not allow for embedded (image) data.
     // Since embedded data is no longer deemed to be a threat, validation can be skipped.
     // See https://github.com/jupyterlab/jupyterlab/issues/5183
-    allowedSchemesAppliedToAttributes: ['href', 'cite']
+    allowedSchemesAppliedToAttributes: ['href', 'cite'],
+    allowedSchemes: []
```

Comment:
```suggestion
    allowedSchemes: ['http', 'https', 'ftp', 'mailto', 'tel']
```

(also see https://github.com/jupyterlab/jupyterlab/pull/13341/files#r1014741922).

Reply:
Awesome nearly done! Thanks for making those changes. In this case we are going to extend off the defaults and restrict user configurability if I understand correctly (pushed my changes below).

In terms of the note in [extension migration guide](https://github.com/jupyterlab/jupyterlab/blob/master/docs/source/extension/extension_migration.rst) not sure exactly what level of detail is needed to explain the changes made to the default sanitizer (as they were quite a lot of changes but we want to keep the note short).